tests/ballerina-unit-test/src/test/resources/test-src/types/xml/xml_iteration.bal

```diff
@@ -113,3 +115,36 @@ function chainedIterableOps() returns xml[] {
                                                 });
     return authors;
 }
+
+xml theXml = xml `<book>the book</book>`;
+xml bitOfText = xml `bit of text\u2702\u2705`;
+xml compositeXml = theXml + bitOfText;
+
+xml xdata = xml `<p:person xmlns:p="foo" xmlns:q="bar">
+        <p:name>bob</p:name>
+        <p:address>
+            <p:city>NY</p:city>
+            <q:country>US</q:country>
+        </p:address>
+        <q:ID>1131313</q:ID>
+    </p:person>`;
+
+function xmlElemIter() returns string {
```

Comment:
Shall we change this to `xmlSequenceIter` and the other to `xmlCharItemIter` (in the Java test methods too)?